picasso/src/main/java/com/squareup/picasso/Downloader.java

```diff
@@ -46,9 +45,9 @@
     final boolean localCacheOnly;
     final int responseCode;
 
-    public ResponseException(String message, boolean localCacheOnly, int responseCode) {
+    public ResponseException(String message, int networkPolicy, int responseCode) {
```

Comment:
Thanks for prompt answer!
By API you mean RequestCreator#networkPolicy? Isn't Downloader public API as well?